test/src/tests/generated/body-complex/models.go

```diff
@@ -55,9 +55,12 @@ type ArrayWrapper struct {
 // Basic
```

Comment:
Should we have here also something like '...' or  `Basic type definition` so its a valid comment ?

Reply:
I can add "..." for this case.

Reply:
Fixed.